src/atlassian/jira.py

```diff
@@ -99,3 +99,24 @@ def get_plugin_pending_task_info(self, task_id: str) -> dict:
             raise ValueError(msg)
 
         return res_json
+
+    def remove_plugin(self, plugin_key: str):
+        """
+        Delete plugin
```

Comment:
Please use [Google style docstring](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)

Reply:
Fixed